ClosedXML/Excel/Ranges/XLRangeBase.cs

```diff
@@ -1090,26 +1072,26 @@ private IXLRangeColumns InsertColumnsBeforeInternal(Boolean onlyUsedCells, Int32
 
             if (!onlyUsedCells)
```

Comment:
Check exactly, what does `onlyUsedCell` supposed to do here.

Reply:
Inserting and shifting of whole columns is rather inconsistent across the codebase. In some places, the column is shifted before this method is called and thus the we can't shift column properties again. In others, the code relies on this method.

Should be consistent, but one small steps at a time.